samples/boards/reel_board/mesh_badge/src/mesh.c

```diff
@@ -76,7 +78,67 @@ BT_MESH_HB_CB_DEFINE(hb_cb) = {
 	.recv = heartbeat,
 };
 
+static void cfg_cli_app_key_status_cb(struct bt_mesh_cfg_cli *cli, uint16_t
+				addr, uint8_t status, uint16_t net_idx,
+				uint16_t app_idx)
+{
+	struct bt_mesh_cfg_cli_mod_pub pub = {
+		.addr = GROUP_ADDR,
+		.app_idx = APP_IDX,
+		.ttl = DEFAULT_TTL,
+		.period = BT_MESH_PUB_PERIOD_SEC(10),
+	};
+	int err;
+
+	/* Bind to vendor model */
+	err = bt_mesh_cfg_cli_mod_app_bind_vnd(NET_IDX, addr, addr, APP_IDX,
+				MOD_LF, BT_COMP_ID_LF, NULL);
+	if (err) {
+		printk("Failed to bind to vendor model (err %d)", err);
+	}
+
+	err = bt_mesh_cfg_cli_mod_app_bind(NET_IDX, addr, addr, APP_IDX,
+				 BT_MESH_MODEL_ID_GEN_ONOFF_SRV, NULL);
+	if (err) {
+		printk("Failed to bind to vendor model (err %d)", err);
+	}
+
+	err = bt_mesh_cfg_cli_mod_app_bind(NET_IDX, addr, addr, APP_IDX,
+				 BT_MESH_MODEL_ID_SENSOR_SRV, NULL);
+	if (err) {
+		printk("Failed to bind to vendor model (err %d)", err);
+	}
+
+	/* Bind to Health model */
+	err = bt_mesh_cfg_cli_mod_app_bind(NET_IDX, addr, addr, APP_IDX,
+				 BT_MESH_MODEL_ID_HEALTH_SRV, NULL);
+	if (err) {
+		printk("Failed to bind to Health Server (err %d)", err);
+	}
+
+	/* Add model subscription */
+	err = bt_mesh_cfg_cli_mod_sub_add_vnd(NET_IDX, addr, addr, GROUP_ADDR,
+				    MOD_LF, BT_COMP_ID_LF, NULL);
+	if (err) {
+		printk("Failed to add subscription (err %d)", err);
+	}
+
+	err = bt_mesh_cfg_cli_mod_pub_set_vnd(NET_IDX, addr, addr, MOD_LF,
+			BT_COMP_ID_LF, &pub, NULL);
```

Comment:
Looks like this line and some other lines above are not aligned correctly.

Reply:
fixed